source/extensions/filters/network/well_known_names.h

```diff
@@ -22,6 +22,8 @@ class NetworkFilterNameValues {
   const std::string HttpConnectionManager = "envoy.http_connection_manager";
   // Mongo proxy filter
   const std::string MongoProxy = "envoy.mongo_proxy";
+  // Mysql proxy filter
+  const std::string MysqlProxy = "envoy.filters.network.mysql_proxy";
```

Comment:
Can this be just `envoy.mysql_proxy`?

Reply:
nope.. There is a comment on top of this file that states the convention